api/openapi-spec/swagger.json

```diff
@@ -19282,7 +19290,7 @@
   },
   "info": {
     "title": "Kubernetes",
-    "version": "v2.17.1"
```

Comment:
revert this version change.

Reply:
didn't change this manually. I'll do another make update.